tests/CookingConvertor/Cooking.java

```diff
@@ -0,0 +1,156 @@
+package CookingConvertor;
+
+
+public class Cooking {
+
+    public static void main(String[] args) {
+        ReceipePrompt prompt = new ReceipePrompt();
+        prompt.run();
+
+    }
+
+    public static class Amounts {
+        private static final float TSP = 5f;
+        private static final float ML = 1f;
+        private static final float OZ = 28f;
+        private static final float POUND= 454f;
+        private static final float GALLON= 3.8f;
+        private static final float QUART= 0.95f;
+        private static final float PINT= 473f;
+        private static final float CUP= 237f;
+        private static final float FL_OUNCE= 30f;
+        private static final float TBSP= 15f;
+
+        public static float value (String unit) {
+            switch (unit) {
+                case Units.TSP:
+                    return TSP;
+                case Units.ML:
+                    return ML;
+                case Units.OZ:
+                    return OZ;
+                case Units.POUND:
+                    return POUND;
+                case Units.GALLON:
+                    return GALLON;
+                case Units.QUART:
+                    return QUART;
+                case Units.PINT:
+                    return PINT;
+                case Units.CUP:
+                    return CUP;
+                case Units.FL_OUNCE:
+                    return FL_OUNCE;
+                case Units.TBSP:
+                    return TBSP;
+                default:
+                return 0;
+            }
+        }
+
+        public static float getTSP() {
+            return TSP;
+        }
+
+        public static float getML() {
+            return ML;
+        }
+
+        public static float getOZ() {
+            return OZ;
+        }
+
+        public static float getPOUND() {
+            return POUND;
+        }
+
+        public static float getGALLON() {
+            return GALLON;
+        }
+
+        public static float getQUART() {
+            return QUART;
+        }
```

Comment:
Remove those and make public static final fields. That is the only good use case for public fields

Reply:
Because you cannot write and all you want is get